### PR TITLE
Ease listing contributors to project visibility

### PR DESCRIPTION
**Pull request by clementbiron**
That changeset set up [All Contributors](https://allcontributors.org/) and use it as a data source for the contributors displayed in the website.

### CODE REVIEW

File: test/cypress/e2e/contributors.cy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  let allContributorsData;
+
+  before(() => {
+    cy.request('contributors.json').then(response => {
+      expect(response.status).to.eq(200);
+      allContributorsData = response.body;
+    });
+  });
+
+  context('when Hugo is started', () => {
```
Comment (Ndpnt) [inline test/cypress/e2e/contributors.cy.js:11]: I do not understand the value of this context

File: test/cypress/e2e/contributors.cy.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+describe('Contributors', () => {
```
Comment (Ndpnt) [inline test/cypress/e2e/contributors.cy.js:1]: I need a sync call to understand the value of these tests; it's not clear for me.

File: .all-contributors.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+This list of contributors is generated automatically by [All Contributors](https://allcontributors.org/). The output used in production is the JSON format, this file is not used.
```
Comment (Ndpnt) [inline .all-contributors.md:1]: Can this file be removed if it is not used?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -70,3 +70,38 @@ We add this additional rule:
 ### CSS naming convention
 
 Use [BEM methodology](https://getbem.com/) to create reusable components and facilitate code sharing.
+
+## Add a contributor
+
+Contributors to Open Terms Archive are listed publicly on the website.
```
Comment (Ndpnt) [inline CONTRIBUTING.md:76]:
```suggestion
## List a new contributor in the Open Terms Archive website

We acknowledge the efforts of our contributors by listing them on our website. Depending on whether the contributor has a GitHub account, the process for adding them differs slightly.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -70,3 +70,38 @@ We add this additional rule:
...
+
+### For contributors with a GitHub account
+
+This is made possible by the use of the [All Contributors bot](https://allcontributors.org/docs/en/bot/overview), that enables adding a contributor with a comment on an issue or pull request, without writing code. To do this, please use the [dedicated issue](https://github.com/OpenTermsArchive/opentermsarchive.org/issues/271) on this repository and write a comment with a command like `@all-contributors please add @<username> for <contributions>`.
+
+### For contributors who do not have a GitHub account
```
Comment (Ndpnt) [inline CONTRIBUTING.md:82]:
```suggestion
### For contributors without a GitHub account
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -70,3 +70,38 @@ We add this additional rule:
...
+
+1. Edit the `.all-contributorsrc` file at the root of this repository and use the following format to add a contributor:
```
Comment (Ndpnt) [inline CONTRIBUTING.md:84]:
```suggestion
If the contributor does not have a GitHub account, the repository files will need to be manually updated:
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -70,3 +70,38 @@ We add this additional rule:
...
+
```
Comment (Ndpnt) [inline CONTRIBUTING.md:85]:
```suggestion
Edit the `.all-contributorsrc` file at the root of this repository by adding the contributor's details in the following format within the contributors array:
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -70,3 +70,38 @@ We add this additional rule:
...
+   }
+   ```
+
+2. Add a picture to `assets/images/contributors/` folder which respects the following constraints:
+
+- square format, 420x420px
+- in [optimised](https://imageoptim.com) JPEG
+- slugify the filename and use `jpg` as an extension, for example: `contributor-name.jpg`
+
+### How to add a type of contribution to a contributor
```
Comment (Ndpnt) [inline CONTRIBUTING.md:105]:
```suggestion
### Add a new contribution type to a contributor
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -70,3 +70,38 @@ We add this additional rule:
...
+
+You can do this by editing the `.all-contributorsrc` file or by using the bot with `@all-contributors please add @<already_listed_contributor> for <new_contribution_type>`
```
Comment (Ndpnt) [inline CONTRIBUTING.md:107]:
```suggestion
To add more contribution types to an existing contributor, you can either edit the `.all-contributorsrc` file directly or use the All Contributors bot by posting a comment in the format: `@all-contributors please add @<already_listed_contributor> for <new_contribution_type>`
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -70,3 +70,38 @@ We add this additional rule:
...
+       }
+     ]
+   }
+   ```
+
+2. Add a picture to `assets/images/contributors/` folder which respects the following constraints:
+
+- square format, 420x420px
+- in [optimised](https://imageoptim.com) JPEG
+- slugify the filename and use `jpg` as an extension, for example: `contributor-name.jpg`
```
Comment (Ndpnt) [inline CONTRIBUTING.md:103]:
```suggestion
- Add a contributor's picture in the `assets/images/contributors/` folder which respects the following constraints:
  - Square format, 420x420 pixels.
  - Optimized JPEG format (you can use tools like [Image Optim](https://imageoptim.com) for optimization).
  - Filename should be a slugified version of the contributor's name with a `.jpg` extension (e.g., `contributor-name.jpg`).
```

**Review by Ndpnt — CHANGES_REQUESTED** [review]

File: .all-contributors.md (hunk 3 above)
Comment (clementbiron) [reply .all-contributors.md:1]: Unfortunately not.

File: test/cypress/e2e/contributors.cy.js (hunk 2 above)
Comment (clementbiron) [reply test/cypress/e2e/contributors.cy.js]: Seen in sync call, thanks.

File: test/cypress/e2e/contributors.cy.js (hunk 1 above)
Comment (clementbiron) [reply test/cypress/e2e/contributors.cy.js]: Seen in sync call, thanks.

**Review by Ndpnt — APPROVED** [review]

**Pull request merged**